code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -6,12 +6,19 @@
 	item_state = null	//so the human update icon uses the icon_state instead.
 	ammo_type = list(/obj/item/ammo_casing/energy/electrode)
 
+/obj/item/weapon/gun/energy/hospistol
+	name = "stun pistol"
+	desc = "A high-tech handgun that fires reusable stun cartridges in an internal magazine. The prize of the Head of Security."
+	icon_state = "hospistol"
+	ammo_type = list(/obj/item/ammo_casing/energy/electrode/gun)
+	can_flashlight = 0
+	pin = /obj/item/device/firing_pin/implant/loyalty
+
 /obj/item/weapon/gun/energy/stunrevolver
 	name = "stun revolver"

```

Comment:
If you are going through with this nerf, at least change the name to reflect that. It is not a stun revolver any more, and I assume you changed the meter from yellow to cyan as well?